numba/cuda/tests/cudapy/test_gufunc_scalar.py

```diff
@@ -7,11 +7,12 @@
 import numpy as np
 from numba import guvectorize, cuda
 from numba import unittest_support as unittest
+from numba.tests.support import TestCase
 from numba.cuda.testing import skip_on_cudasim, SerialMixin
 
 
 @skip_on_cudasim('ufunc API unsupported in the simulator')
-class TestGUFuncScalr(SerialMixin, unittest.TestCase):
+class TestGUFuncScalr(SerialMixin, TestCase):
```

Comment:
Whilst this line is being edited, perhaps the `a` could be added `TestGUFuncScalr=>TestGUFuncScalar` ?